src/resources/resources.json

```diff
@@ -171,18 +171,68 @@
-      "cdnResourcesConsolidationDescription": "Dieser Task aktualisiert die Liste der in den Dokumenten und Dokument-Revisionen verknüpften internen Speicher-Ressourcen. Eventuell inkorrekte Daten werden dabei berichtigt.",
-      "cdnDirectoriesCreationHeader": "Upload-Verzichnisse auf dem internen Speicher erstellen",
-      "cdnDirectoriesCreationDescription": "Dieser Task stellt sicher, dass für jedes Dokument in der Datenbank (inclusive privater Dokumente) ein korrespondierendes öffentliches Verzeichnis im internen Speicher existiert. Fehlende Verzeichnisse werden hierbei neu angelegt.",
+      "batchSectionHeader_document-regeneration": "Dokumente neu erstellen",
+      "batchSectionDescription_document-regeneration": "Dieser Task erstellt alle Dokumente in der Datenbank neu, basierend auf den vorhandenen Dokument-Revisionen. Eventuell inkorrekte Dokument-Daten werden dabei berichtigt.",
+      "batchSectionHeader_document-validation": "Dokumente validieren",
+      "batchSectionDescription_document-validation": "Führt eine Validierung der Datenstruktur aller Dokumente und Dokument-Revisionen in der Datenbank durch. Dieser Task ändert keine Daten.",
+      "batchSectionHeader_cdn-resources-consolidation": "Ressourcen auf dem internen Speicher konsolidieren",
+      "batchSectionDescription_cdn-resources-consolidation": "Dieser Task aktualisiert die Liste der in den Dokumenten und Dokument-Revisionen verknüpften internen Speicher-Ressourcen. Eventuell inkorrekte Daten werden dabei berichtigt.",
+      "batchSectionHeader_cdn-upload-directory-creation": "Upload-Verzichnisse auf dem internen Speicher erstellen",
+      "batchSectionDescription_cdn-upload-directory-creation": "Dieser Task stellt sicher, dass für jedes Dokument in der Datenbank (inclusive privater Dokumente) ein korrespondierendes öffentliches Verzeichnis im internen Speicher existiert. Fehlende Verzeichnisse werden hierbei neu angelegt.",
       "lastExecution": "Letzte Ausführung",
       "start": "Task starten"
     }
   },
+  {
+    "namespace": "userAccountsTab",
+    "language": "en",
+    "resources": {
+      "lastLogin": "Last login",
+      "expiryDate": "Expiry date",
+      "accountLocked": "Locked",
+      "roles": "Roles",
+      "providerKey": "Provider key",
+      "externalUserId": "External ID",
+      "connectedUser": "Connected user",
+      "activeAccounts": "Active accounts",
+      "externalAccounts": "External accounts",
+      "accountsWithStorage": "Accounts with storage",
+      "closedAccounts": "Closed accounts",
+      "selectPlan": "Select plan",
+      "addReminder": "Add reminder",
+      "removeAllReminders": "Remove reminders",
+      "deleteAllOwnedRooms": "Delete rooms",
+      "storageSpace": "Storage space",
+      "reminders": "Reminders",
+      "accountClosedOn": "Account closing date",
+      "filterPlaceholder": "Filter by name, email, etc."
```

Comment:
We only can filter by name or email actually. I think we might get a wrong expectation due to the ".etc" part.

Reply:
That was the state before external accounts, now we can also filter by identity provider key and external user ID. I first wanted to change the placeholder depending on the current table the user sees, but then I realized this is very confusing and magical, so I decided to write it like that to indicate we're searching in all "relevant" fields. If the "etc." is too confusing we could also just simply say "Filter results" or something. What do you think?

Reply:
Made a ticket for unifying filters, as discussed https://educandu.atlassian.net/browse/EDU-1080